src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type MoviesProps = {
+  movies?: Movie[];
```

Comment:
Try to fix eslint error 
![image](https://user-images.githubusercontent.com/70009205/222762775-af451725-6cf5-4063-9d0f-d9f59afde208.png)